posts/2022-09-27-instant-on-220011-beta.adoc

```diff
@@ -0,0 +1,1171 @@
+
+The Open Liberty InstantOn feature makes it really easy for you to try it out by providing a new `checkpoint` action. The `checkpoint` action can be executed against your existing Open Liberty server configurations. With the `checkpoint` action you specify a phase at which you want the Liberty startup process to be stopped and a checkpoint of the process saved.  When Liberty is launched it will detect that there is a saved checkpoint process and it will resume the process from the state the process was saved during the `checkpoint` action.
+
+== Setup the example application project
+
+Before getting into the details of how the `checkpoint` action works lets setup a working example. For that we will have a look at the link:/guides/getting-started.html[Getting started with Open Liberty] guide.  If you would like to follow along then start by cloning the link:https://github.com/openliberty/guide-getting-started.git[Git repository] and use the projects that are provided inside:
+[source]
+----
+
+git clone https://github.com/openliberty/guide-getting-started.git
+cd guide-getting-started
+----
+We will work within the `finish/` directory for this demonstration. To try out the application, run the following Maven goal to build the application and deploy it to Open Liberty:
+[source]
+----
+cd finish
+mvn liberty:run
+----
+After you see the following message, your application server is ready:
+[source]
+----
+The defaultServer server is ready to run a smarter planet.
+----
+Check out the service at the http://localhost:9080/dev/system/properties URL.
+
+After you are finished checking out the application, stop the Open Liberty server by pressing CTRL+C in the command-line session where you ran the server.
+
+To build the WAR for the application run the following :
+[source]
+----
+mvn package
+----
+After this you will have a `target/guide-getting-started.war` archive built.  We can now include this WAR into a container image
+
+== Containerizing the application
+To use Open Liberty InstantOn support an application must first be containerized.  For more details on how to containerize your application with Open Liberty first see the link:/guides/containerize.html[Containerizing microservices] guide or the link:/guides/containerize-podman.html[Containerizing microservices with Podman] guide. If you are unfamiliar with Podman then it is recommended to follow the Podman guide first because the Open Liberty Instant On support currently requires the use of Podman.  Support for Docker should come later once a Docker release is made that supports the necessary capabilities that checkpoint and restore require.
```

Comment:
This almost makes it sound like you have to do one of these guides and bring your own containerized app to this demo- but that's not the case is it? Might be good to clarify that

Reply:
I'm going to clarify this.